packages/notifi-wallet-provider/lib/context/NotifiWallets.tsx

```diff
@@ -73,7 +73,11 @@ export const NotifiWalletProvider: React.FC<PropsWithChildren> = ({
 
   useEffect(() => {
     const storageWallet = getWalletsFromLocalStorage();
-    if (storageWallet) {
+    if (
+      storageWallet &&
+      selectedWallet &&
+      Object.keys(wallets).includes(selectedWallet)
```

Comment:
Do we have to have these two extra condition?

Reply:
yes, if not adding      
` selectedWallet && Object.keys(wallets).includes(selectedWallet)`
`wallets[storageWallet.walletName]` will get error as, the wallet name is not in wallets if we connect injective wallet

Reply:
I see, this looks like a kind of a patch workaround tho. 
Shall we add a comment saying the purpose of these two lines and also todo to remove that